acos_client/v30/axapi_http.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2014,  Doug Wiegley,  A10 Networks.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import json
+import logging
+
+import requests
+
+try:
+    import http.client as http_client
+except ImportError:
+    # Python 2
+    import httplib as http_client
```

Comment:
If you want you could handle this by checking the version

```
import sys
if sys.version_info >= (3, 0):
    import http.client as http_client
else
    import httplib as http_client
```

That way if there is a problem with http.client unrelated to our code, the correct error would be shown. But either way is reasonable enough to me.